lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -225,9 +225,9 @@ func (c *ContainerProxy) AddVolumesToContainer(handle string, config types.Conta
 		return handle, derr.NewErrorWithStatusCode(fmt.Errorf("%s", err), http.StatusBadRequest)
 	}
 
-	volumeSubset, err := processSpecifiedVolumes(config.HostConfig.Binds)
-	if err != nil {
-		return handle, derr.NewErrorWithStatusCode(fmt.Errorf("%s", err), http.StatusBadRequest)
+	volumeSubset, processErr, createErr := processSpecifiedVolumes(config.HostConfig.Binds)
```

Comment:
Thanks @fdawg4l! I'm currently refactoring the `AddVolumesToContainer` func so I don't need to return multiple errors from `processSpecifiedVolumes` while still keeping track of the errors returned from `VolumeCreate`.